renovate.json

```diff
@@ -18,6 +18,16 @@
       "major": {"enabled": false},
       "minor": {"enabled": false}
     },
+    {
+      "description": "Group patch-only updates to develop weekly to cut down on PR noise.",
+      "groupName": "Lock-file Patch Releases for LTM branches",
+      "matchBaseBranches": ["ltm/*"],
+      "schedule": [
+        "before 4am on Friday"
+      ],
+      "major": {"enabled": false},
+      "minor": {"enabled": false}
+    },
```

Comment:
We discussed before about hard-pinning our dependencies so as no surprise upgrades affect this but I think that might be too hard of a line and our already restrictive requirements has shown problematic for sensible use.

Since we won't be planning bi-weekly releases of our LTM, should we instead move this check to a monthly update?

We might need to find an alternative to Dependabot (or maybe this configuration option: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#target-branch) to get security updates about our LTMs.